src/pages/library/RequestDatasetAccessModal.js

```diff
@@ -0,0 +1,83 @@
+import _ from 'lodash/fp'
+import { useState } from 'react'
+import { div, h, span, table, tbody, td, th, thead, tr } from 'react-hyperscript-helpers'
+import { ButtonPrimary, IdContainer } from 'src/components/common'
+import { TextArea } from 'src/components/input'
+import Modal from 'src/components/Modal'
+import { Ajax } from 'src/libs/ajax'
+import { withErrorReporting } from 'src/libs/error'
+import { FormLabel } from 'src/libs/forms'
+import { cond, useCancellation, withBusyState } from 'src/libs/utils'
+
+
+export const RequestDatasetAccessModal = ({ onDismiss, datasets }) => {
+  const [reason, setReason] = useState('')
+
+  return h(Modal, {
+    title: 'Request Access',
+    width: '40rem',
+    showCancel: false,
+    onDismiss
+  }, [
+    div([`
+      You cannot access this dataset because it is protected by an Authorization Domain.
+      You need to obtain permission from the owner(s) of this dataset in order to get access.
+      Clicking the "Request Access" button below will send an email to the admins of that dataset.`]),
```

Comment:
```suggestion
    div([
      'You cannot access this dataset because it is protected by an Authorization Domain. ',
      'You need to obtain permission from the owner(s) of this dataset in order to get access. ',
      'Clicking the "Request Access" button below will send an email to the admins of that dataset.'
    ]),
```